app/decorators/user_decorator.rb

```diff
@@ -106,4 +106,9 @@ def fully_banished?
   def stackbit_integration?
     access_tokens.any?
   end
+
+  def considered_new?
```

Comment:
I deliberated for a bit if I really already want to add this, but Peter already hinted at more rate limits of a similar nature and it did make the code in `CommentsController` and `Article::Creator` quite a bit nicer so I went for it.